src/Routes/SamplePage/SamplePage.js

```diff
@@ -1,15 +1,11 @@
 import React from 'react';
-import { withRouter, Link } from 'react-router-dom';
-import { useDispatch } from 'react-redux';
-
-import { Button, StackItem, Stack, Title } from '@patternfly/react-core';
-import { Main, PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
-import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/';
-
-import asyncComponent from '../../Utilities/asyncComponent';
-const SampleComponent = asyncComponent(() => import('../../Components/SampleComponent/sample-component'));
-
+import { withRouter } from 'react-router-dom';
+import { Button, Card, CardBody } from '@patternfly/react-core';
+import { Section, Main, PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
```

Comment:
Can we structure the files like this since we are moving to SmartComponent: `src/SmartComponents/DriftPage/RosPage.js ` like here -> https://github.com/RedHatInsights/drift-frontend/pull/35/files